custom_model_runner/datarobot_drum/resource/components/Python/uwsgi_component/uwsgi_serving.py

```diff
@@ -124,11 +124,17 @@ def load_model_callback(self, model_path, stream, version):
 
     @FlaskRoute("{}/".format(os.environ.get(URL_PREFIX_ENV_VAR_NAME, "")), methods=["GET"])
     def ping(self, url_params, form_params):
-        return HTTP_200_OK, {"message": "OK"}
+        if self._error_response:
+            return HTTP_513_DRUM_PIPELINE_ERROR, self._error_response
+        else:
+            return HTTP_200_OK, {"message": "OK"}
 
     @FlaskRoute("{}/ping/".format(os.environ.get(URL_PREFIX_ENV_VAR_NAME, "")), methods=["GET"])
     def ping2(self, url_params, form_params):
-        return HTTP_200_OK, {"message": "OK"}
+        if self._error_response:
+            return HTTP_513_DRUM_PIPELINE_ERROR, self._error_response
+        else:
+            return HTTP_200_OK, {"message": "OK"}
```

Comment:
nit (for consistency with the rest of the code and being more pythonic):
```suggestion
        return HTTP_200_OK, {"message": "OK"}
```